sproul-club/sc-frontend
# SCF-8: Remove "Invite-only events are hidden." text

(SCF-8)[https://scfrontend.atlassian.net/browse/SCF-8?atlOrigin=eyJpIjoiZGU2MTI3NmIzYzAzNGZkMjk1YzM2YTY4NTExNzc2NjAiLCJwIjoiaiJ9]: Remove "Invite-only events are hidden." 

Steps to test:
1. Verify on student dashbord that after "*Only public events displayed." it doesn't say "Invite-only events are hidden."
2. The onboarding modal isn't visible in the develop branch right now, only main, but if it WERE visible, you would verify that after "Note: Only public events displayed." it doesn't say "Invite-only events are hidden."

<img width="536" alt="image" src="https://user-images.githubusercontent.com/16441620/137549524-033e3160-f665-495f-9dd6-a8e23e8f987a.png">
